bot/pkg/bsky/bsky.go

```diff
@@ -92,3 +104,17 @@ func extractRKey(uri string) (string, error) {
 
 	return parts[len(parts)-1], nil
 }
+
+func createPost(URI string, content string, postType post.NYTContentType) (post.Post, error) {
```

Comment:
It took me a while to get what's happening here, I think because of we're turning a `bsky.Post` into a `post.Post`. Suggestion only -- Is there a way to make that clearer perhaps by renaming this function slightly?

Reply:
I had a similar experience when looking through this code. I went with `createPost` to be consistent with a similar function in `mastadon.go`. I think you are right though, it could be more clear. Maybe `bskyPostToPost`?